### DISCUSSION TITLE
How to trigger completion on key press?

### DISCUSSION THREAD

**Discussion by rstanuwijaya** [marlonrichert/zsh-autocomplete#383] [Q&A] [answered]
Hi, I'm a new user, and thanks for making such a fantastic extension. 

While the extension is handy, I don't need the suggestion dropdown most of the time. Is it possible to trigger the dropdown manually by pressing some keys?

Moreover, for the directory suggestion, is it possible to hide the folders that start with a dot?

Thanks!

**Comment by marlonrichert** [comment, marked as answer]
> Hi, I'm a new user, and thanks for making such a fantastic extension.

Glad to hear. You're welcome! 🤗 

> While the extension is handy, I don't need the suggestion dropdown most of the time. Is it possible to trigger the dropdown manually by pressing some keys?

Yes, add the following to your `.zshrc` file _before_ sourcing the plugin:
```zsh
zstyle ':autocomplete:async' enabled no
```

> Moreover, for the directory suggestion, is it possible to hide the folders that start with a dot?

It wasn't, but I've now changed it so that dotfiles are hidden by default and shown only when the shell option `GLOB_DOTS` has been set (which is off by default): #384.

**Reply by marlonrichert** [reply]
PS: If you enjoy using my software, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂

**Comment by nopunYntendeed** [comment]
The plugin stops behaving like described from [aed8e17](https://github.com/marlonrichert/zsh-autocomplete/commit/4c363ff553be227db90842d7481fd6e02de8703c) onwards.
Just shout out if I this is an actual issue
(async got way snappier :100: )

**Reply by marlonrichert** [reply]
Which part of the description?
